CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [unreleased]
+### Security
+### Added
+### Changed
+### Removed
+
+## [v0.7.0](https://github.com/flatcar-linux/locksmith/releases/tag/v0.7.0)- 29/11/2021
+
+### Security
+- Go dependencies update to latest versions ([#6](https://github.com/flatcar-linux/locksmith/pull/6), [#8](https://github.com/flatcar-linux/locksmith/pull/8))
+
+### Changed
+- ignore nil sent by dbus.Signal in the daemon ([#1](https://github.com/flatcar-linux/locksmith/pull/1))
+- import path to `flatcar-linux` ([#2](https://github.com/flatcar-linux/locksmith/pull/2))
+- Go modules to handle dependencies ([#4](https://github.com/flatcar-linux/locksmith/pull/4))
+- node will not reboot outside of reboot window in case lock is acquired outside it ([#10](https://github.com/flatcar-linux/locksmith/pull/10))
+
+### Fixed
+- panic on shutdown ([#1](https://github.com/flatcar-linux/locksmith/pull/1))
+
```

Comment:
Maybe something like this?
```suggestion
### Security
- Update Go dependencies to latest versions ([#6](https://github.com/flatcar-linux/locksmith/pull/6), [#8](https://github.com/flatcar-linux/locksmith/pull/8))

### Changed
- Use Go modules to handle dependencies ([#4](https://github.com/flatcar-linux/locksmith/pull/4))
- Use `github.com/flatcar-linux/locksmith` as Go module name ([#2](https://github.com/flatcar-linux/locksmith/pull/2))
- Node will not reboot outside of reboot window in case lock is acquired outside of it ([#10](https://github.com/flatcar-linux/locksmith/pull/10))

### Fixed
- Possible panic on shutdown ([#1](https://github.com/flatcar-linux/locksmith/pull/1))

```

I'd also consider putting dots at the end of each line.

Reply:
Yeah, that's good - let's commit this. 

> I'd also consider putting dots at the end of each line.

I don't really care - but here's my PoV: we can have a dot if we consider the entry + the title as a whole sentence: "Changed to Go modules to handle dependencies." In our case, it's more a list: "Here's what changed: * ..." so no dot to me.

Reply:
Sure, no dot then!

Reply:
It should be good now. :)